packages/cli/README.md

```diff
@@ -334,9 +347,175 @@ ARGUMENTS
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:add STREAMID`
+
+pin a stream
+
+```
+USAGE
+  $ glaze pin:add STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be pinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
```

Comment:
what does `--key` do on a pin request?

Reply:
Nothing, it's just that the flag is set for all commands (just like `--ceramic`) to simplify the logic and reduce code duplication in individual commands code.
The docs for these commands are automatically generated but we could edit this manually if we really need to.

Reply:
how many commands is it actually relevant for?  If its a small number (like if it's just the create and update commands), maybe it should be added to those commands explicitly and not in a way that makes it show up as an arg for all commands?

Reply:
It's related to other logic I'd rather not change, so I think we can remove it from the commands docs rather than change the implementation.